src/ui/setup/dynamic/devtools.ts

```diff
@@ -206,7 +206,7 @@ export default async function setupDevtools(store: AppStore, replayClient: Repla
     );
   });
 
-  await setupApp(store, replayClient);
+  setupApp(store, replayClient);
```

Comment:
This `await` wasn't doing anything before (because the `setupApp` method was not blocking on the async methods it called). Updating that method to block on what it called caused a dead lock here, so I removed this `await`.

In general, our initialization code here (and in a few other places) is pretty fragile. I'd like for us to rethink it at some point, but later.